test_suite/cloud/test_aws.py

```diff
@@ -558,12 +558,25 @@ def test_yum_group_install(self, host):
             pytest.skip('Not applicable to Atomic host AMIs')
 
         with host.sudo():
-            assert host.run_test('yum -y groupinstall "Development tools"'), \
-                'Error while installing Development tools group'
-
-            package_to_check = 'glibc-devel'
-            assert host.package(package_to_check).is_installed, \
-                f'{package_to_check} is not installed'
+            attempts = 2
+            for attempt in range(attempts):
```

Comment:
As there will always be only two attempts, I would remove this for loop so we can reduce nesting.